docs/explanations/document-storage.md

```diff
@@ -0,0 +1,154 @@
+# Bluesky Document Storage
+
+## Executive Summary
+
+For the first time in the ten-year history of the Bluesky project, the Bluesky
+core developers will soon recommend a change in how data and metadata from
+Bluesky documents should be stored.
```

Comment:
I'd prefer an absolute time like `in the first quarter of 2025` over `will soon recommend`.

Reply:
Or maybe refer to the last paragraph of this document.